tidb-in-kubernetes/scale-tidb.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 扩缩容 TiDB 集群
+category: how-to
+---
+
+ # 水平扩缩容
+
+ 水平扩缩容操作指的是通过增加或减少节点的数量，来达到集群扩缩容的目的。
+
+ 扩缩容 TiDB 集群时，会按照填入的 replicas 值，对 PD、TiKV、TiDB 进行顺序扩缩容操作。扩容操作按照节点编号由小到大增加节点，缩容操作按照节点编号由大到小删除节点。
+
+ ## 扩缩容操作
+
+1. 修改集群的 `value.yaml` 文件中的 `pd.replicas`、`tidb.replicas`、`tikv.replicas` 至期望值；
+2. 执行 `helm upgrade` 命令进行扩缩容：
+
+     {{< copyable "shell-regular" >}}
+
+     ```shell
```

Comment:
please fix the alignments (this one and all others in the following)

Reply:
done